### ISSUE TITLE
Delete whole trash folder

### ISSUE THREAD

**Issue by d3hof** [nextcloud/mail#4286]
### Feature Request

Delete the whole trash folder

#### Summary

Make it possible to delete the whole trash folder with one button.

**Comment by StCyr** [comment]
Why not implementing a 'Delete all' button to delete all messages from a (any) mailbox (the 'trash folder" being a mailbox like another)?

**Comment by d3hof** [comment]
you're right that would be even nicer to have. It is already possible to mark mails by hand. Nice would be if you can just mark all and then delete them.

**Comment by guettli** [comment]
Related: "Select all" (then you can select all, and then delete them): https://github.com/nextcloud/mail/issues/7880

**Comment by madbob** [comment]
Fixed in 396fe08
